core/templates/dev/head/pages/landing/fractions/landing_page_teacher.html

```diff
@@ -335,54 +368,113 @@ <h2 class="oppia-fractions-landing-h2 oppia-fractions-landing-centered-h2" style
     .oppia-fractions-landing-image {
       position: relative;
       z-index: 20;
-      max-width: 80%;
+      max-width: 100%;
       max-height: 500px;
     }
     .oppia-fractions-landing-image-mobile {
       display: none;
     }
+    .video-frame {
```

Comment:
Could you please prefix this class name with .oppia-fractions-landing-... ? Ditto for anything below this one as well (text-box, easy-to-follow, etc.) -- want to make sure the CSS is well-scoped and only applies to this page.